src/Agent/NewRelic.Api.Agent/NewRelic.cs

```diff
@@ -642,7 +642,7 @@ public static void StartAgent()
         /// </code>
         /// </example>
         [MethodImpl(MethodImplOptions.NoOptimization | MethodImplOptions.NoInlining)]
-        public static void SetApplicationName(string applicationName, string applicationName2 = null, string applicationName3 = null)
+        public static void SetApplicationName(string? applicationName, string? applicationName2 = null, string? applicationName3 = null)
```

Comment:
Makes sense the that the last two parameters are nullable, but why do we want to enable the first one to be nullable?

Reply:
I unresolved this conversation because I wanted to double check... did you intentionally leave the first `applicationName` as nullable?